pages/use-case/human-resource.vue

```diff
@@ -53,18 +56,32 @@ export default {
     SeamlessIntegration,
     BlogCard,
   },
-  async asyncData({ $content }) {
-    let articles = await $content('blog')
-      .where({
-        published: true,
-        featured: false,
-      })
-      .search('human resource')
-      .sortBy('createdAt', 'desc')
-      .fetch()
+  async asyncData() {
+    const {
+      data: { data },
+    } = await axios.get(
+      `${process.env.strapiUrl}/api/blogs`,
+      {
+        params: {
+          'filters[$or][0][title][$contains]': 'HR',
+          'filters[$or][1][title][$containsi]': 'human resource',
+          sort: 'publishedAt:desc',
+          populate: '*',
+        },
+      }
+    )
+
+    let articles = data.map((item) => {
+      return {
+        id: item.id,
+        ...item.attributes,
+        coverImg: item.attributes.coverImg.data.attributes.url,
+        readingStats: readingTime(item.attributes.body),
```

Comment:
We might need to document the `attributes` somewhere?

Reply:
Sorry, I didn't get it..

Reply:
Yes, we can be okay with it.